### DISCUSSION TITLE
getting 🚫 ope: Sign in to confirm you’re not a bot

### DISCUSSION THREAD

**Discussion by Flamango121** [codetheweb/muse#1174] [Q&A]
i only have a youtube api and get the error 🚫 ope: Sign in to confirm you’re not a bot when i try to play anything

**Comment by PythonTilk** [comment]
you figgured it out?

**Reply by Mubelotix** [reply]
Just sign it

**Reply by PythonTilk** [reply]
> Just sign it

How? Where tf do I need to sign shit?

**Reply by Mubelotix** [reply]
Rtfm https://github.com/museofficial/muse?tab=readme-ov-file#running

**Reply by PythonTilk** [reply]
did everything. same shitty error @Mubelotix

**Reply by Mubelotix** [reply]
Youtube might have blacklisted your IP then. Are you reusing an IP someone might have been using to run Invidious?

**Reply by PythonTilk** [reply]
Im using a vps but can change the ip
